web/styles/subscriptions.css

```diff
@@ -364,6 +364,19 @@ h4.user_group_setting_subsection_title {
             display: flex;
             justify-content: space-between;
         }
+
+        .nothing-to-show {
+            display: block;
+            margin-top: calc(45vh - 75px);
+            text-align: center;
+            font-size: 1em;
+            margin-left: 2em;
+            margin-right: 2em;
+
+            span {
+                color: hsl(0, 0%, 67%);
```

Comment:
We should avoid duplicating CSS here as the text in right side panel also has same CSS.